requirements.txt

```diff
@@ -6,5 +6,6 @@ jaxlib>=0.3.0,<=0.3.2
 jax>=0.3.0,<=0.3.4
 flax>=0.4.0
 bm3d
+bm4d
```

Comment:
Should be added to `NOCONDA` list in `make_conda_env.sh`.

Reply:
Added it.